### PR TITLE
 chore: escape checkbox in PR template

### PR DESCRIPTION
**Pull request by TieWay59**
## Brief Information

This pull request is in the type of ([more info](https://github.com/angular/angular/blob/main/CONTRIBUTING.md#type) about types):

- `[ ]` build
- `[ ]` ci
- `[x]` docs
- `[ ]` feat
- `[ ]` fix
- `[ ]` perf
- `[ ]` refactor
- `[ ]` test

Related issues ([all available keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)):

- #734

## Details

Just avoid GitHub PR regex on the checkbox in the most straightforward way.

## Checklist

- `[x]` I have read the [CONTRIBUTING](https://github.com/hypertrons/hypertrons-crx/blob/master/CONTRIBUTING.md) doc
- `[x]` I have signed the [CLA](https://cla-assistant.io/hypertrons/hypertrons-crx)

## Others

N.A.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,28 +1,47 @@
-## Brief Information
+<!-- 
+🚨NOTE🚨 Please complete the following tasks before submitting a pull request:
 
-This pull request is in the type of ([more info](https://github.com/angular/angular/blob/main/CONTRIBUTING.md#type) about types):
+- [ ] Read the CONTRIBUTING doc. see: https://github.com/hypertrons/hypertrons-crx/blob/master/CONTRIBUTING.md
+- [ ] Sign the CLA. see: https://cla-assistant.io/hypertrons/hypertrons-crx
+-->
```
Comment (tyn1998) [inline .github/pull_request_template.md:6]: I think these are true tasks. [These awesome pr templates](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) use checkboxes in this way. So personally I prefer making them as checkboxes rather comments.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,28 +1,47 @@
...
+- `ci`: Changes related to continuous integration, such as configuration files and scripts.
+- `docs`: Changes related to documentation, such as README files and API documentation.
+- `feat`: New features or enhancements to existing features.
+- `fix`: Bug fixes or patches to existing features.
+- `perf`: Changes related to performance improvements.
+- `refactor`: Changes that improe code quality or structure without changing functionality.
+- `test`: Changes related to testing, such as adding or modifying unit tests. 
 
-- keyword #xxx
+Ref: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#type
```
Comment (tyn1998) [inline .github/pull_request_template.md:21]: Descriptions for types are really good except for a typo "improe" :)

I'm afraid  that `This pull request is in the type of:` is tend to be missed by newcomers. And another fact is that in practice a PR can belong to multiple types, but people don't like typing too much, so we resorted to checkboxes before.

Maybe we can figure out a better way.

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -1,28 +1,47 @@
...
+Example:
+
+    This pull request is in the type of: build
+-->
+
+Related issues:
+<!-- 
+Link the issue to the PR.
+
+Ref: https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword
```
Comment (tyn1998) [inline .github/pull_request_template.md:32]: In my veiw, ref links are better in the form of `[description](url)` and to be placed in the content rather in the comment. In that way, GitHub newcomers who cannot edit the PR description to see the markdown source are able to visit the references easily (just click the link on the text), so can learn these tips by reading existing PRs  before actually creating their first PR in this repo.

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -1,28 +1,47 @@
...
-## Checklist
+Example:
 
-- [ ] I have read the [CONTRIBUTING](https://github.com/hypertrons/hypertrons-crx/blob/master/CONTRIBUTING.md) doc
-- [ ] I have signed the [CLA](https://cla-assistant.io/hypertrons/hypertrons-crx)
+    Related issues:
+
+    - fix #xx1
+    - address #xx2 (mention but not close)
+-->
```
Comment (tyn1998) [inline .github/pull_request_template.md:40]: How about this way:

```text
Related issues:

- close #111 (mentnion and link)
- close #222
- #333 (just mention)
```

**Review by tyn1998 — CHANGES_REQUESTED** [review]
Hey @TieWay59, thanks for your first PR in Hypercrx! I have reviewed your PR carefully and created several review comments for you. I'm sorry to know that yml is not applicable to PR yet, so if there is no better way to provide a "chooser" the checkboxes is still a choice.

File: .github/pull_request_template.md (hunk 3 above)
Comment (TieWay59) [reply .github/pull_request_template.md:32]: Newcomers still need to check the preview to get the link. And I believe most newcomers don't how to edit the PR text after posting or checking the preview carefully.

It also seems verbose if it includes much irrelevant text in the output post. But it's OK if you can take that.

File: .github/pull_request_template.md (hunk 4 above)
Comment (TieWay59) [reply .github/pull_request_template.md:40]: Sure, I see the `address` is used in other communities, it's not a keyword, but means to handle something. But shorter for better, it's also nice.

**Comment by TieWay59** [comment]
@tyn1998 That's it. Here comes the most simple way to fix the problem and keep everything else as you'd prefer.

**Comment by tyn1998** [comment]
Well, `[x]` looks just too weird 😆 for me.

image

Would you mind keeping those checkboxes for a while longer? We may wait GitHub to release yaml pr templates then fix #734.

**Pull request closed without merging**